suite-native/module-trading/src/hooks/history/useChangeStringsExtractor.ts

```diff
@@ -38,32 +32,16 @@ export const useChangeStringsExtractor = (
         if (value === undefined || cryptoId === undefined) {
             return undefined;
         }
-        const { coinSymbol } = cryptoIdToSymbolAndContractAddress(cryptoId);
-
-        let formattedValue;
-        if (coinSymbol && isNetworkSymbol(coinSymbol)) {
-            const { decimals } = getNetwork(coinSymbol);
-
-            formattedValue = CryptoAmountFormatter.format(value, {
-                maxDisplayedDecimals: decimals,
-                isBalance: true,
-                symbol: coinSymbol,
-                isEllipsisAppended: false,
-                smallestUnitsOverride,
-            });
-
-            const splitValue = formattedValue.split(' ');
-            if (splitValue.length > 1) {
-                formattedValue = `${formatNumberWithThousandCommas(splitValue[0])} ${splitValue.slice(1).join(' ')}`;
-            } else if (splitValue.length > 0) {
-                formattedValue = formatNumberWithThousandCommas(splitValue[0]);
-            }
-        } else {
-            const decimalValue = convertTokenValueToDecimal(value, 0);
-            formattedValue = `${localizeNumber(decimalValue)} ${coinSymbol?.toLocaleUpperCase() || ''}`;
-        }
-
-        return formattedValue;
+        const coinSymbol = cryptoIdToCoinSymbol(cryptoId);
+        const networkDecimals = coinSymbol ? getNetworkDecimals(coinSymbol) : undefined;
+
+        return CryptoAmountFormatter.format(value, {
+            maxDisplayedDecimals: networkDecimals ?? TOKEN_DECIMALS_LENGTH,
```

Comment:
This is simpler IMO 🤔 
```suggestion
        const decimals = coinSymbol ? getNetworkDecimals(coinSymbol) : TOKEN_DECIMALS_LENGTH;

        return CryptoAmountFormatter.format(value, {
            maxDisplayedDecimals: decimals,
```

Reply:
Good idea, but that would unfortunately won't work for case when `coinSymbol` is defined and `getNetworkDecimals()` returns undefined. In that case the `maxDisplayedDecimals` would get `undefined` instead of intended `TOKEN_DECIMALS_LENGTH`.